app/static/src/app/store/run/state.ts

```diff
@@ -15,6 +16,22 @@ export interface ParameterSet {
     hidden: boolean
 }
 
+type AdConfStandardForm = {
+    val: [number|null, number|null],
+    defaults: number[],
```

Comment:
I'd probably go for 'default' rather than 'defaults' since 'val' is singular... and can't the type be [number, number]?

Reply:
done changed prop to default and changed type to [number, number]